### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ requirements:
     - r-knitr
     - r-magrittr
     - imagemagick
-    - librsvg
+    - librsvg 2.44.*
```
Comment (bgruening) [inline recipe/meta.yaml]: Narf, do you know why that is? It seems to be in the global pinnings :( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L414

Comment (dpryan79) [reply recipe/meta.yaml]: It's getting ignored and 2.40.* is getting installed. I don't know why that's happening, but that's the problem.

Comment (dpryan79) [reply recipe/meta.yaml]: Ah

    conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
      - librsvg=2.44
      - r-base=3.5.1

Comment (dpryan79) [reply recipe/meta.yaml]: Aha, r-base needs to be rebuilt with the current cairo pinning!

Comment (bgruening) [reply recipe/meta.yaml]: That makes more sense, but more complicated and r-base will probably fail :(

Comment (dpryan79) [reply recipe/meta.yaml]: This was when it changed: https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/665c45c430fd214466c51049f855113d9e91737b

Comment (dpryan79) [reply recipe/meta.yaml]: I'll make a PR and we'll see how it goes.

**Pull request closed without merging**